static/redfish.yaml

```diff
@@ -3731,6 +3731,7 @@ definitions:
       UserName:
         description: This property contains the user name for the account.
         type: string
+        pattern: '^[A-Za-z][A-Za-z0-9._-]+$'
```

Comment:
This pattern isn't the same with the monorail-2.0.yaml. I don't find a reason to keep them different, please align them.